Makefile

```diff
@@ -68,6 +68,13 @@ ifeq ($(UNAME_S),Darwin)
 	ICONV := tools/iconv.py UTF-8 SHIFT-JIS
 endif
 
+WSL := 0
+JAVA := java
+ifeq ($(findstring microsoft,$(shell cat /proc/sys/kernel/osrelease)),microsoft)
```

Comment:
I'd prefer we use this method to detect whether we're on windows: https://stackoverflow.com/questions/714100/os-detecting-makefile (the above if chain is from here too). The WSL and JAVA vars can be declared above too I think

Reply:
We're not detecting Windows, we're detecting _WSL_